.github/workflows/rune_with_docker_and_crictl.yml

```diff
@@ -0,0 +1,222 @@
+name: Run rune with docker and crictl test
+
+# Controls when the action will run. Triggers the workflow on push or pull request.
+on: [push, pull_request]
+
+jobs:
+  # TODO: add rune circtl test on ubuntu image.
+  rune_ubuntu:
+    runs-on: self-hosted
+    steps:
+    - uses: actions/checkout@v1
+
+    - name: Get Occlum version
+      run: wget https://raw.githubusercontent.com/occlum/occlum/master/src/pal/include/occlum_version.h;
+        echo "OCCLUM_VERSION=$(grep 'Version =' occlum_version.h | awk '{print $4}')" >> $GITHUB_ENV;
+        echo "RUNE_VERSION=$(grep 'Version:' rune/dist/rpm/rune.spec | awk '{print $2}')" >> $GITHUB_ENV
+
+    - name: Create container
+      run: docker rm -f ubuntu || true;
+        docker run -itd --privileged --rm --name=ubuntu --net host --device /dev/isgx -v /var/run/aesmd:/var/run/aesmd -v $GITHUB_WORKSPACE:/root/inclavare-containers occlum/occlum:$OCCLUM_VERSION-ubuntu18.04
+
+    - name: Install dependencies
+      run: docker exec ubuntu bash -c "apt-get update;
+        apt-get install libseccomp-dev;
+        cd /root && wget https://dl.google.com/go/go1.14.2.linux-amd64.tar.gz;
+        tar -zxvf /root/go1.14.2.linux-amd64.tar.gz -C /usr/lib;
+        echo 'export GOROOT=/usr/lib/go' >> /etc/profile;
+        echo 'export GOPATH=/root/gopath' >> /etc/profile;
+        echo 'export PATH=$GOROOT/bin:$PATH:$GOPATH/bin' >> /etc/profile;
+        echo 'export GOPROXY="https://mirrors.aliyun.com/goproxy,direct"' >>/etc/profile;
+        echo 'export PKG_CONFIG_PATH=/usr/lib/pkgconfig' >>/etc/profile;
+        echo 'export GO111MODULE=on' >>/etc/profile;
+        source /etc/profile;
+        go get github.com/golang/protobuf/protoc-gen-go@v1.3.5"
+
+    - name: Build and install rune
+      run: docker exec ubuntu bash -c "cd /root;
+        cp -r inclavare-containers inclavare-containers-$RUNE_VERSION;
+        tar zcf v$RUNE_VERSION.tar.gz inclavare-containers-$RUNE_VERSION;
+        pushd /root/inclavare-containers-$RUNE_VERSION;
+        echo "$RUNE_VERSION" > VERSION;
+        sed -i '17 d' rune/dist/deb/build.sh;
+        sed -i '17icp /root/v*.tar.gz \$DEBBUILD_DIR' rune/dist/deb/build.sh;
+        source /etc/profile;
+        chmod +x rune/dist/deb/build.sh && make package components=rune;
+        dpkg -i rune_$RUNE_VERSION-1_amd64.deb;
+        popd"
```

Comment:
inclavare-containers-$RUNE_VERSION is not used any more?

Reply:
yes, we use inclavare-containers-$RUNE_VERSION to create deb package.